plugins/view-resources/src/components/CreateAttribute.svelte

```diff
@@ -0,0 +1,106 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import core, { Class, generateId, PropertyType, Ref, Space, Type } from '@anticrm/core'
+  import presentation, { getClient } from '@anticrm/presentation'
+  import { AnyComponent, EditBox, DropdownLabelsIntl, Label, Component, Button } from '@anticrm/ui'
+  import { DropdownIntlItem } from '@anticrm/ui/src/types'
+  import { createEventDispatcher } from 'svelte'
+  import view from '../plugin'
+  import { getEmbeddedLabel } from '@anticrm/platform'
+
+  export let _class: Ref<Class<Space>>
+  let name: string
+  let type: Type<PropertyType> | undefined
+  let is: AnyComponent | undefined
+  const client = getClient()
+  const hierarchy = client.getHierarchy()
+  const dispatch = createEventDispatcher()
+
+  async function save (): Promise<void> {
+    if (type === undefined) return
```

Comment:
Please show confirmation dialog and inform about changes will not be possible to perform for type and field name in document.